backend-diagram-converter/core/src/main/java/org/camunda/community/migration/converter/expression/ExpressionTransformationResult.java

```diff
@@ -19,4 +22,16 @@ public String getNewExpression() {
   public void setNewExpression(String newExpression) {
     this.newExpression = newExpression;
   }
+
+  public Boolean hasMethodInvocation() {
+    Pattern pattern = Pattern.compile("\\.[\\w]*\\(.*\\)");
```

Comment:
Instead of dynamic calculation, all data contained in the pojo could be immutable fields.

Reply:
changed.